code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -392,18 +412,17 @@ GLOBAL_LIST_INIT(brass_recipes, list ( \
 	throw_range = 3
 	turf_type = /turf/open/floor/clockwork
 	novariants = FALSE
-	grind_results = list("iron" = 5, "teslium" = 15)
+	grind_results = list("brass" = 20)
+
+/obj/item/stack/tile/brass/interact(mob/user, recipes_sublist)
+	if(is_servant_of_ratvar(user))
```

Comment:
Shouldn't this be !is_servant_of_ratvar(user) ?

Reply:
fuck